plutus-chain-index/plutus-chain-index.cabal

```diff
@@ -108,18 +108,17 @@ executable marconi
     -threaded -rtsopts -with-rtsopts=-N -Wall -Wcompat
     -Wincomplete-uni-patterns -Wincomplete-record-updates
     -Wno-missing-import-lists -Wredundant-constraints -O0
+    -Wunused-packages
```

Comment:
Good catch.

Reply:
It's very unreliable, I need to `rm -rf dist-newstyle/build/x86_64-linux/ghc-8.10.7/plutus-chain-index-0.1.0.0` to double check.

Reply:
I think it would be a good idea to add that to our CONTRIBUTING.md as an extra check, or something to that effect to keep our dependencies in line.